docs/src/getting-started.rst

```diff
@@ -10,43 +10,53 @@
 
 .. _getting_started:
 
-#############
-Hello, |cdk|!
-#############
+#####################################
+Creating Your First |cdk| Application
+#####################################
 
 This topic walks you through creating and deploying your first |cdk| app.
 
 .. _setup:
 
-Setup
-=====
+Setting up the |cdk|
+====================
 
 .. _setup_prerequisites:
 
 Prerequisites
 -------------
 
-`Node.js (>= 8.11.x) <https://nodejs.org/en/download>`_ - required for the
-command-line toolkit and language bindings.
+You must install
+`Node.js (>= 8.11.x) <https://nodejs.org/en/download>`_ to use the command-line
+toolkit and language bindings.
+
+You must set the `JAVA_HOME` environment variable to the path to where you have
```

Comment:
I would start the sentence with "If you use Java...", so people can "bail out" early if they don't care

Reply:
will do